src/popover.scss

```diff
@@ -102,6 +102,11 @@ $block: #{$fd-namespace}-popover;
       z-index: $fd-popover-z-index + 1;
     }
 
+    // Inner shadows can affect outer shadow. This will discard such behavior
+    > * {
+      overflow: hidden;
+    }
```

Comment:
We can't use this solution unfortunately. We have cases where the size of the popover body is limited and the scroll should appear, but if we add this solution the scroll no longer works. Please check "Scrollable" example of Popover. https://deploy-preview-2296--fundamental-styles.netlify.app/?path=/docs/components-popover--alignment#scrollable

The scroll is not working, if you remove the `overflow: hidden;` it will appear.